### ISSUE TITLE
Fetch employee data job failing with TypeError on platform

### ISSUE THREAD

**Issue by daissatou2** [OpenFn/fca#17]
## Describe the bug
The [job](https://github.com/OpenFn/fca/blob/staging/fetchEmployees.js) to fetch employee data from Sympa HR is failing with the following error: TypeError [Error]: Cannot convert undefined or null to object.

This job was working when we handed it off for UAT a few weeks ago. 

## To Reproduce
1. Here is a link to a failed run on OpenFn.org which is indicative of the bug: https://openfn.org/projects/fca-sympa-hr-salesforce-staging/runs/0646ab08-0b34-7cac-ae32-3887876e1d6f

### expression.js
Link to the job itself in Github: https://github.com/OpenFn/fca/blob/staging/fetchEmployees.js

### state.json

```json
{
  "configuration": { "SEE LAST PASS: `FCA Sympa HR API`"},
  "data": NA
  "cursor": "Fetch employees where "DataCardModified" >= date of last sync"
}
```

## Expected behavior
This job successfully fetches employees from Sympa 

## To test/resolve
1. After the desired output is working locally (from the CLI), please [push commits to staging].
2. [Please test the change on OpenFn.org by re-running this run ([link](https://openfn.org/projects/fca-sympa-hr-salesforce-staging/runs/0646ab08-0b34-7cac-ae32-3887876e1d6f)) and confirming success.]

## Toggl 
`FCA SympaHR - Salesforce [2023]`

**Comment by aleksa-krolls** [comment]
@daissatou2 is this still an issue? 
@mtuchi pls do not work on this if you cannot replicate.

**Comment by mtuchi** [comment]
@daissatou2 i have fixed this bug on this PR #18, Please merge and test on platform-app 
cc @aleksa-krolls
